pkg/didcomm/common/service/service.go

```diff
@@ -20,16 +20,24 @@ type DIDComm interface {
 	Event
 }
 
-// DIDCommMsg did comm msg
+// DIDCommMsg is used by handlers/dispatchers/events to pass DIDComm messages.
 type DIDCommMsg struct {
+	// message type
+	Type string
+	// message
+	Payload []byte
+}
+
+// Msg is used for state machine processing.
+type Msg struct {
```

Comment:
Perhaps we should have a TODO for splitting inbound and outbound type.

Reply:
Created an issue #490 and added a [TODO](https://github.com/hyperledger/aries-framework-go/pull/482/files#diff-75342fbf986feea5a6bb6a33d0caff76R32).

Reply:
Updated the changes in this PR.